[2025-06-27T04:24:59.785+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T04:24:59.859+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt_v1.0.start_task scheduled__2025-03-03T05:00:00+00:00 [queued]>
[2025-06-27T04:24:59.892+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt_v1.0.start_task scheduled__2025-03-03T05:00:00+00:00 [queued]>
[2025-06-27T04:24:59.893+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T04:24:59.955+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): start_task> on 2025-03-03 05:00:00+00:00
[2025-06-27T04:24:59.995+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1103) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T04:25:00.014+0000] {standard_task_runner.py:72} INFO - Started process 1146 to run task
[2025-06-27T04:25:00.052+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt_v1.0', 'start_task', 'scheduled__2025-03-03T05:00:00+00:00', '--job-id', '273', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt.py', '--cfg-path', '/tmp/tmpdv0jciyv']
[2025-06-27T04:25:00.061+0000] {standard_task_runner.py:105} INFO - Job 273: Subtask start_task
[2025-06-27T04:25:00.333+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt_v1.0.start_task scheduled__2025-03-03T05:00:00+00:00 [running]> on host f79cc72a155d
[2025-06-27T04:25:00.580+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt_v1.0' AIRFLOW_CTX_TASK_ID='start_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-03T05:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-03T05:00:00+00:00'
[2025-06-27T04:25:00.595+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T04:25:00.602+0000] {logging_mixin.py:190} INFO - start
[2025-06-27T04:25:00.608+0000] {logging_mixin.py:190} INFO - ds: 2025-03-03
[2025-06-27T04:25:00.609+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-06-27T04:25:00.648+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T04:25:00.650+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt_v1.0, task_id=start_task, run_id=scheduled__2025-03-03T05:00:00+00:00, execution_date=20250303T050000, start_date=20250627T042459, end_date=20250627T042500
[2025-06-27T04:25:00.709+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T04:25:00.840+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-27T04:25:00.855+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
